docs/content/changelogs/2023-11-27-42.7.1-release.md

```diff
@@ -0,0 +1,24 @@
+---
+title:  PostgreSQL JDBC Driver 42.6.1 Released
+date:   2023-11-27 10:05:33 -0500
+categories:
+    - new_release
+version: 42.7.1
+---
+**Notable changes**
+
+
```

Comment:
WDYT of adding high-level overview of the changes?
I mentioned known issues in 42.7.0 release notes (we probably need to back-patch them to the site so jdbc.postgresql.org shows known issues as well): https://github.com/pgjdbc/pgjdbc/releases/tag/REL42.7.0

I suggest adding (right after "notable changes" and before the low-level changelog) something like

Fixes issues introduced in 42.7.0:
- NoSuchMethodError on ByteBuffer#position When Running on Java 8  when accessing arrays, issue https://github.com/pgjdbc/pgjdbc/issues/3014
- SET followed by SHOW in one statement returns wrong result, issue https://github.com/pgjdbc/pgjdbc/issues/3007
- 42.7.0 updated `DateStyle` from `ISO` to `ISO, MDY` (it aligns with PostgreSQL defaults), and it result in wrong results when server uses non-default `DateStyle` (see https://github.com/pgjdbc/pgjdbc/issues/3008)

Reply:
Yes, will fix